src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -22,48 +24,30 @@ public LineService(LineRepository lineRepository) {
 
     public LineResponse saveLine(LineRequest request) {
         Line line = lineRepository.save(new Line(request.getName(), request.getColor()));
-        return new LineResponse(
-                line.getId(),
-                line.getName(),
-                line.getColor(),
-                line.getCreatedDate(),
-                line.getModifiedDate()
-        );
+        return LineResponse.of(line);
     }
 
     @Transactional(readOnly = true)
     public List<LineResponse> findAllLines() {
         List<Line> lines = lineRepository.findAll();
 
         return lines.stream()
-            .map(this::createLineResponse)
+            .map(line -> LineResponse.of(line))
```

Comment:
이런 정적 메서드 호출도 메소드 레퍼런스를 이용할 수 있습니다 😄 
```suggestion
            .map(LineResponse::of)
```